content/applications/marketing/surveys/dashboard.rst

```diff
@@ -0,0 +1,178 @@
+.. image:: dashboard/three-dot-dropdown.png
+   :align: center
+   :alt: The three-dot drop-down menu of options that appears on the Odoo Surveys dashboard.
+
+The options are:
+
+- :guilabel:`Edit Survey`: when clicked, Odoo reveals the survey form for that particular survey,
+  which can then be modified in a number of different ways.
+- :guilabel:`Share`: when clicked, Odoo reveals the :guilabel:`Share a Survey` pop-up form that can
+  be used to invite potential participants to the survey.
+- :guilabel:`Delete`: when clicked, Odoo presents a pop-up window, wherein the user **must** confirm
+  they want to delete the survey entirely, which they can do by clicking the :guilabel:`Delete`
+  button at the bottom of the pop-up window.
+- :guilabel:`Color`: users can opt to choose a color to add to the survey line on the dashboard for
+  added organizational purposes, if needed.
+
+Beneath the buttons that are located to the far-right of the survey lines, there is an *Activities*
+button, represented by a :guilabel:`🕘 (clock)` icon. When clicked, a mini pop-up window appears,
+from which activities related to that particular survey can be scheduled and customized.
+
+.. image:: dashboard/schedule-activities-dropdown.png
+   :align: center
+   :alt: The Schedule Activities drop-down menu that appears on the Odoo Surveys dashboard.
+
+List view
+---------
+
+The *Surveys* dashboard is shown in the Kanban view, by default, but there is also a list view
+option available in the upper-right corner, represented by a :guilabel:`≣ (bars)` icon.
+
+When the :guilabel:`≣ (bars)` icon is clicked, the survey related data is displayed in a list view.
+
+.. image:: dashboard/list-view.png
+   :align: center
+   :alt: The list view option located on the Odoo Surveys application dashboard.
+
+The columns shown on the *Surveys* app dashboard, while in list view, are as follows:
+
+- :guilabel:`Survey Title`
+- :guilabel:`Responsible`
+- :guilabel:`Average Duration`
+- :guilabel:`Registered`
+- :guilabel:`Success Ratio (%)`
+- :guilabel:`Avg Score (%)`
+
+.. tip::
+   Additional columns can be added to the *Surveys* application dashboard, while in list view, by
+   clicking the *additional options* drop-down menu, located to the far-right of the column titles,
+   represented by a :guilabel:`(slider with two dots)` icon.
+
+Activities view
+---------------
+
+To have the *Surveys* application dashboard display nothing but the activities associated to the
+surveys in the database, click the :guilabel:`🕘 (clock)` icon to the far-right of the other view
+options, located in the upper-right corner.
+
+.. image:: dashboard/activities-view.png
+   :align: center
+   :alt: The activities view option located on the Odoo Surveys dashboard.
+
+Doing so reveals a table with rows and columns. The rows show the different surveys in the database,
+and the columns depict the various activity types.
+
+.. important::
+   The :guilabel:`New` button does **not** appear in this view.
```

Comment:
I don't feel that this is _important_ information, rather it is just a note. It may be helpful to note the use-case of the view rather than just what _cannot_ be done while on this view
```suggestion
.. note::
   A new survey cannot be created in this view, as it is solely for the purpose of creating and
   viewing scheduled activities.
```